apps/ui-kit/src/lib/styles/index.css

```diff
@@ -121,7 +121,8 @@
 
     /* SECONDARY TEXT */
     --secondary-text-color: theme('colors.iota-neutral.40');
-    --secondary-text-error-color: theme('colors.iota-error.30');
+    --secondary-text-error-color: theme('colors.iota-error.20');
+    --secondary-error-bg-color: theme('colors.iota-error.90');
```

Comment:
`--secondary-error-bg-color` is not a necessary variable, you can use `--error-surface` for backgrounds, as in other components :pray: 
```
light: --error-surface: theme('colors.iota-error.90');
dark: --error-surface: theme('colors.iota-error.10');
names: --error-surface: theme('colors.names-error.20');
```